src/app/shared/components/input-chip/input-chip.component.html

```diff
@@ -21,8 +21,7 @@
       <ion-label>{{ chip.label }}</ion-label>
     </ion-chip>
     <ion-chip *ngIf="showAddOption" (click)="addNewOption()">
-      <ion-icon name="add" *ngIf="!selectedChips.has(chip)"></ion-icon>
-      <ion-icon name="close-circle" color="white" *ngIf="selectedChips.has(chip)"></ion-icon>
+      <ion-icon name="add"></ion-icon>
```

Comment:
is this change required?

Reply:
yes on click add only need to show alert selectedchip for add is not required

Reply:
I think we need add and close icons. @rejneesh1  Please confirm

Reply:
@kiranharidas187  this should be fine , its removed for add chip (which always need to  show add icon only)